infra/checkout.bicep

```diff
@@ -1,104 +1,53 @@
-@minLength(1)
-@maxLength(64)
-@description('Name of the the environment which is used to generate a short unqiue hash used in all resources.')
-param name string
-
-@minLength(1)
-@description('Primary location for all resources')
 param location string
-
+param nameseed string
```

Comment:
```suggestion
param nameSeed string
```

Reply:
removed nameseed and replaced with resourceToken